---
title: FileCorruptedException
second_title: Aspose.Email for .NET API 参考
description: 初始化FileCorruptedExceptionaspose.email/filecorruptedexception类.
type: docs
weight: 10
url: /zh/net/aspose.email/filecorruptedexception/filecorruptedexception/
---
## FileCorruptedException() {#constructor}

初始化[`FileCorruptedException`](../../filecorruptedexception)类.

```csharp
public FileCorruptedException()
```

### 也可以看看

* class [FileCorruptedException](../../filecorruptedexception)
* 命名空间 [Aspose.Email](../../filecorruptedexception)
* 部件 [Aspose.Email](../../../)

---

## FileCorruptedException(string) {#constructor_1}

初始化[`FileCorruptedException`](../../filecorruptedexception)具有指定错误消息的类。

```csharp
public FileCorruptedException(string message)
```

| 范围 | 类型 | 描述 |
| --- | --- | --- |
| message | String | 描述错误的消息。 |

### 也可以看看

* class [FileCorruptedException](../../filecorruptedexception)
* 命名空间 [Aspose.Email](../../filecorruptedexception)
* 部件 [Aspose.Email](../../../)

---

## FileCorruptedException(string, Exception) {#constructor_2}

初始化[`FileCorruptedException`](../../filecorruptedexception)具有指定错误消息的类。

```csharp
public FileCorruptedException(string message, Exception inner)
```

| 范围 | 类型 | 描述 |
| --- | --- | --- |
| message | String | 描述错误的消息。 |
| inner | Exception | 导致当前异常的异常。 |

### 也可以看看

* class [FileCorruptedException](../../filecorruptedexception)
* 命名空间 [Aspose.Email](../../filecorruptedexception)
* 部件 [Aspose.Email](../../../)

<!-- DO NOT EDIT: generated by xmldocmd for Aspose.Email.dll -->
